client/app/bundles/course/material/folderLoader.ts

```diff
@@ -0,0 +1,15 @@
+import { LoaderFunction, redirect } from 'react-router-dom';
+import { getIdFromUnknown } from 'utilities';
+
+import CourseAPI from 'api/course';
+
+const folderLoader: LoaderFunction = async ({ params }) => {
+  const folderId = getIdFromUnknown(params?.folderId);
+  if (!folderId) return redirect('/');
```

Comment:
Does this redirect to the root of the application? (i.e. https://coursemology.org/)
If it does, should it redirect to the root folder of the workbin instead